.circleci/config.yml

```diff
@@ -33,7 +33,9 @@ jobs:
           command: |
             make clean
             make test
-
+      - run:
+          name: Run htmlproofer (no external links)
+          command: bundle exec scripts/htmlproofer --internal
```

Comment:
The `--internal` flag means we will only check internal links. We've found external ones can be a little flaky.